Assets/Scripts/PlayerFall.cs

```diff
@@ -10,60 +10,57 @@ public class PlayerFall : MonoBehaviour
     //speed at which the player is dragged down
     public float thrust = 10.0f;
     private GameObject Camera;
+    private GameObject TrapDoor;
     private GameObject Player;
     //height from Player we want Camera to be
     private float desiredCameraHeight = 5.0f;
     private Vector3 direction;
     private float timer = 0.0f;
     private float delayuntilFastFall = .5f;
     
-    //falling audio at this time overriden in BeginFall method
     [SerializeField]
     private AudioSource fallingAudio;
     
     //flag
-    public bool trapOpened {get; set;}
+    private bool trapOpened = false;
     
     void Start(){
         
-        GameObject.Find("TrapDoor").transform.GetChild(0).transform.GetChild(0).GetComponent<TrapDoorOpen>().TrapOpened += BeginFall;
-        GameObject.Find("TrapDoor").transform.GetChild(1).transform.GetChild(0).GetComponent<TrapDoorOpen>().TrapOpened += BeginFall;
+        Camera = GameObject.FindWithTag("MainCamera");
+        TrapDoor = gameObject;
+        TrapDoorOpen[] hinges = GetComponentsInChildren<TrapDoorOpen>();
+        foreach(TrapDoorOpen script in hinges)
+        script.TrapOpened += BeginFall;
+        
     }
     
-    void BeginFall(){
+    void BeginFall(GameObject fallingEntity){
+        
         direction = new Vector3(0, desiredCameraHeight, 0);
-        Player = GameObject.Find("MainPlayer");
-        PlayerBod = Player.GetComponent<Rigidbody>();
-        fallingAudio = GameObject.FindWithTag("Boss").GetComponents<AudioSource>()[1];
+        PlayerBod = fallingEntity.GetComponent<Rigidbody>();
+        Player = fallingEntity;
+        AudioSource[] audios = GameObject.FindWithTag("Boss").GetComponentsInChildren<AudioSource>();
+        foreach(AudioSource asource in audios)
+        if(asource.clip.name == "Jes£s Lastra - Cries From Hell")
+        fallingAudio = asource;

```

Comment:
Not only do you not put curly braces, but you don't even indent this line. This is illegal in some countries for a reason 😡 The fact that this might work is beyond me.

Reply:
Fixed lol don't want to get deported